examples/multi-modal-data-movielens/02-Data-Enrichment.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2021 NVIDIA Corporation. All Rights Reserved.\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License.\n",
+    "# =============================================================================="
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://developer.download.nvidia.com/compute/machine-learning/frameworks/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# MovieLens Data Enrichment\n",
+    "\n",
+    "In this notebook, we will enrich the MovieLens 25M dataset with poster and movie sypnopsis scrapped from IMDB.\n",
+    "\n",
+    "First, we will need to install some extra package for IMDB data collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "wget -O master.zip https://codeload.github.com/alberanid/imdbpy/zip/master\n",
+    "unzip master.zip\n",
+    "cd imdbpy-master && python setup.py install"
```

Comment:
It looks like `imdbpy` is available from PyPI with a release in late April. Could this be accomplished with `pip install imdbpy`?

Reply:
great, easy fix, thanks

Reply:
fixed